apps/linkedin-to-notion/src/components/LinkedInNotionSidePanel/Form.tsx

```diff
@@ -0,0 +1,150 @@
+    setFirstName(firstName);
+    setLastName(lastName);
+    setJobTitle(jobTitle);
+    setCurrentCompany(currentCompany);
+    setLocation(location);
+  };
+
+  return (
+    <div className="flex flex-col space-y-3">
+      <SelectEntry
+        labelText="Status"
+        id="status"
+        handleChange={(e) => setStatus(e.target.value)}
+        initialValue={status}
+        options={[
+          {
+            id: 'not-contacted',
+            label: 'Not contacted',
+            value: 'notContacted',
+          },
+          {
+            id: 'contacted',
+            label: 'Contacted',
+            value: 'contacted',
+          },
+          {
+            id: 'in-process',
+            label: 'In process',
+            value: 'inProcess',
+          },
+          {
+            id: 'no-match',
+            label: 'No match',
+            value: 'noMatch',
+          },
+          {
+            id: 'not-interested',
+            label: 'Not interested',
+            value: 'notInterested',
+          },
+          {
+            id: 'hired',
+            label: 'Hired',
+            value: 'hired',
+          },
+        ]}
+      />
+      <div className="flex space-x-4">
+        <TextEntry
+          initialValue={firstName}
+          placeholder="Guy"
+          inputId="linkedin-first-name"
+          handleChange={(e) => setFirstName(e.target.value)}
+          labelText="First Name"
+        />
+        <TextEntry
+          initialValue={lastName}
+          placeholder="Tarenbois"
+          inputId="linkedin-last-name"
+          handleChange={(e) => setLastName(e.target.value)}
+          labelText="Last Name"
+        />
+      </div>
+      <TextEntry
+        initialValue={jobTitle}
+        placeholder="Musicien"
+        inputId="linkedin-job-title"
+        handleChange={(e) => setJobTitle(e.target.value)}
+        labelText="Job title"
+      />
+      <TextEntry
+        initialValue={currentCompany}
+        placeholder="Rock Band"
+        inputId="linkedin-current-company"
+        handleChange={(e) => setCurrentCompany(e.target.value)}
+        labelText="Current company"
+      />
+      <TextEntry
+        initialValue={location}
+        placeholder="Paris"
+        inputId="linkedin-location"
+        handleChange={(e) => setLocation(e.target.value)}
+        labelText="Location"
+      />
+      <SelectEntry
+        labelText="Gender"
+        id="gender"
+        handleChange={(e) => setGender(e.target.value)}
+        initialValue={gender}
+        options={[
+          {
+            id: 'empty-gender',
+            label: '',
+            value: '',
+          },
+          {
+            id: 'female',
+            label: 'Female',
+            value: 'female',
+          },
+          {
+            id: 'male',
+            label: 'Male',
+            value: 'male',
+          },
+          {
+            id: 'other',
+            label: 'Other',
+            value: 'other',
+          },
```

Comment:
![image](https://github.com/Bastien-and-Gauvain/monorepo/assets/45038783/d0b97a38-9215-46b0-943c-8cef25e4ab8e)